events-api/src/modules/tickets/dto/create-ticket.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsNotEmpty, IsNumber, IsString } from "class-validator";
+
+export class CreateTicketDto {
+    @ApiProperty({
+        example: 5
+    })
+    @IsNumber()
```

Comment:
Add better validation. Can the amount of tickets be -1? Or can we order 1.5 ticket?